feat: adds reactive Map class to svelte/reactivity (#10803)
sveltejs/svelte: sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

> Objects and arrays passed to `$state.frozen` will be shallowly frozen using `Object.freeze()`. If you don't want this, pass in a clone of the object or array instead.

### Reactive Map, Set and Date

> Svelte provides reactive wrappers for the native `Map`, `Set` and `Date` objects. These can be found imported from `svelte/reactivity` and used just like their native equivalents.

Reviewer: ```suggestion
Svelte provides reactive `Map`, `Set` and `Date` classes. These can be imported from `svelte/reactivity` and used just like their native counterparts.
```
